src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -30,7 +31,8 @@ public Phone(String phone) {
      * Returns true if a given string is a valid phone number.
      */
     public static boolean isValidPhone(String test) {
-        return test.matches(VALIDATION_REGEX);
+        String trimmedTest = test.trim();
```

Comment:
I think the point of doing the exact math was to make sure there is no leading/trailing whitespace right?